pkg/querier/blocks_store_queryable.go

```diff
@@ -135,6 +137,11 @@ func newBlocksStoreQueryableMetrics(reg prometheus.Registerer) *blocksStoreQuery
 			Name: "cortex_querier_blocks_with_compactor_shard_but_incompatible_query_shard_total",
 			Help: "Blocks that couldn't be checked for query and compactor sharding optimization due to incompatible shard counts.",
 		}),
+		// Named to be consistent with distributor_query_ingester_chunks_total
```

Comment:
Nit: the metric is named `cortex_distributor_query_ingester_chunks_total` (with `cortex_` prefix).

Reply:
This is really a nit, not worth a PR, but if you address the other comment then you could fix this too.